src/app/dialog/dialog.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="dialog">
+  <div class="dialog__photo">
+    <img class="dialog__img" [src]="data.photo" [alt]="data.name" />
+  </div>
+  <div class="dialog__content">
+    <div class="dialog__title">
+      <h1 class="dialog__h1">{{ data.name }}</h1>
+      <button mat-mini-fab class="dialog__close" (click)="close()">X</button>
+    </div>
+    <div class="dialog__description">
+      <h2>Main characteristics:</h2>
+      <p class="dialog__p">Brand: {{ data.brand | titlecase }}</p>
+      <p class="dialog__p">Color: {{ data.color | titlecase }}</p>
```

Comment:
`dialog__p` doesn't show sense of element. Not needed to show element tag in class